### PR TITLE
Enable creating rooms

### PR DESCRIPTION
**Pull request by alexbaldwin**

### CODE REVIEW

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -69,5 +69,17 @@ def show
       end
     end
     @comments = @message.comments.where(now_playing: false, aired_at: nil).order(created_at: :asc)
+    @new_message = Message.new
```
Comment (alexbaldwin) [inline app/controllers/messages_controller.rb]: Should I kill this?

Comment (ianhirschfeld) [inline app/controllers/messages_controller.rb]: Yes, this is no longer needed

**Comment by ianhirschfeld** [comment]
Do you guys want to enable this @alexbaldwin @wr? This PR is ready to go if you are.

**Comment by wr** [comment]
:shipit:

**Pull request merged**